docs/review-merge-process.md

```diff
@@ -0,0 +1,23 @@
+# Mappings Review / Merge Process
+
+This document details the review and merge process for Parchment mappings.
+
+### Requirements to merge:
+
+- If we have any automatic tests, they all must pass.
+  - Example: A PR may not be merged if the Contributor License Agreement (CLA) is not signed.
+- A PR requires two approving reviews
+  - One approving review is required to be from a Parchment team member.
+- We ask that people don’t review things that they don’t actually have the capability to effectively review (knowledge wise).
+- All external reviewers are welcome, but not required.
```

Comment:
> Who will the second review be from, if not from a Parchment team member?

Anyone who wants to review it.

> Do external reviews count towards the two-approvals requirement?

Yes, that is why two are required, but only one is mentioned as required to be from Parchment.

Reply:
Fair enough, though discussion in Discord has given thought to allowing small contribution PRs to only require one approving review instead of at least two.